governance.md

```diff
@@ -0,0 +1,149 @@
+
+## Decision making
+
+Coraza is a meritocratic, consensus-based community project.
+
+### Team members
+
+Team member status may be given to those who have made ongoing contributions to the Coraza project for at least 2 months. This is usually in the form of significant code improvements and/or notable work on documentation.
+
+New members may be proposed by any existing member in the monthly meeting. It is highly desirable to reach consensus about acceptance of a new member. However, the proposal is ultimately voted on by a formal [majority vote](#majority-vote).
+
+If the new member proposal is accepted, the proposed team member will be pinged in a github issue on this repository and confirm its acceptance.
+
+Team members may retire at any time by opening a github issue in this repository requesting for removal.
+
+Team members can be removed by [majority vote](#majority-vote) opening a github issue and voting in the next 48 hours.
+For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Any removal vote can cover only one single person.
+
+In case a member leaves, the [offboarding](#offboarding) procedure is applied.
+
+The current team members can be found under this link.
+
+### Code owners
+
+Code owners lead one or more project(s) or parts thereof and serve as a point of conflict resolution amongst the contributors to this project. Ideally, code owners are also team members, but exceptions are possible for suitable code owners that, for whatever reason, are not yet team members.
+
+Changes in code ownership have to be announced in the `#coraza-core` channel. They are decided by [rough consensus](#consensus) and formalized by changing the `CODEOWNERS` file of the respective repository in a PR.
+
+Code owners are granted commit rights to all projects covered by this governance.
+
+A code owner or committer may resign by notifying the team. A code owner with no project activity for a year is considered to have resigned. Code owners that wish to resign are encouraged to propose another team member to take over the project. A project with no code owners might be considered to be archived.
+
+A project may have multiple code owners, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
+
+### Administrators
+
+Administrators are team member with access to manage the [Coraza organization](https://github.com/corazawaf). Any change in the organization should be agree upon vote in the `#coraza-core` channel. The role of administrator does not represent any kind of hierarchy among team members.
+
+For the sake of independence, no two administrators can work for the same company despite the regime (full time, consultancy, etc).
```

Comment:
If three people are deciding, then two of them should not have the same background. 

But if more people are in the mix, then the number must be different. I think the idea is that if `N` persons if the size of the core, then at most `(N-1)/2` people can share the same background, and take over the decisions of the global community.